migration/vault_item_import.py

```diff
@@ -204,9 +227,14 @@ def migrate_items(csv_data):
             }] if otp_secret else [])
 
         if not template:
-            logging.warning(f"No template generated for the {title} item. Skipping...")
+            print(f"\t\"{title}\" => skipped (incompatible item)")
+            stats["skipped"] += 1
             continue
 
         json_template = json.dumps(template)
         vault_to_use = created_vault_list[vault] if vault_defined else personal_vault['id']
         create_item(vault_to_use, json_template)
+        stats["migrated"] += 1
+        print(f"\t\"{title}\" => migrated")
+    
+    print(f"\nMigration complete! Total {stats['total']} credentials. Migrated {stats['migrated']} credentials, created {stats['vaults']} vaults, skipped {stats['skipped']} credentials.")
```

Comment:
Would this look better if each stat is put in a separate line?

Reply:
Changed